### PR TITLE
Numalogic original payload support

### PR DESCRIPTION
**Pull request by sarabala1979**

### CODE REVIEW

File: prometheus-pusher/main.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func (p *prometheusSink) Sink(ctx context.Context, datumStreamCh <-chan sinksdk.
 		failed = failed.Append(sinksdk.ResponseFailure(datum.ID(), "failed to push the metrics"))
 	}
 	var pls []PrometheusPayload
-	var prometheusPayload PrometheusPayload
+	var prometheusPayloads []*PrometheusPayload
 	for _, payloadMsg := range payloads {
 		p.metrics.IncreaseTotalPushed()
 		if p.enableMsgTransformer {
```
Comment (sarabala1979) [inline prometheus-pusher/main.go:121]: Having a feature flag to support univariate payload  and multivariate numalogic payload

Comment (ab93) [reply prometheus-pusher/main.go:121]: This is just temporary right?

Comment (sarabala1979) [reply prometheus-pusher/main.go:121]: yes. once we moved to Multivariant. we can remove this code

**Review by ab93 — APPROVED** [review]
Rubber stamp

**Pull request merged**